$NetBSD: patch-Makefile,v 1.3 2019/11/24 20:21:23 adam Exp $

Do not build npm-index.html on rebuild.
Do not build htmldocs, as these need some binary packages.

--- Makefile.orig	2022-03-03 21:10:57 UTC
+++ Makefile
@@ -21,6 +21,7 @@ files_mandocs = $(shell find docs/conten
                |sed 's|docs/content/configuring-npm/|man/man5/|g' ) \
 
 misc_mandocs = $(shell find docs/content/using-npm -name '*.md' \
+               \! -name 'npm-index.md' \
                |sed 's|.md|.7|g' \
                |sed 's|docs/content/using-npm/|man/man7/|g' ) \
 
@@ -28,7 +29,7 @@ mandocs = $(cli_mandocs) $(files_mandocs
 
 all: docs
 
-docs: mandocs htmldocs
+docs: mandocs
 
 # don't regenerate the snapshot if we're generating
 # snapshots, since presumably we just did that.
